### DISCUSSION TITLE
Unable to run Postal

### DISCUSSION THREAD

**Discussion by calybre** [postalserver/postal#2918] [Help with using Postal]
Hi everyone!

Just for the record I'm not well versed in deploying applications but I can follow instructions. 

I've tried to install Postal a couple of times now without any success(!sic). Usually ends in Connection timeout when trying to access my-domain.com to login to Postal.

My last approach ended as usual with en error "Web server is down" on cloudflare (I manage there DNS records).

I bought a VPS to run test installations (**specs lower than recommened** - might this be the issue?)

Ubuntu 22. I follow everything as in the installation guide. Docker, git, MariaDB. 

Everything goes smooth until I run "postal initialize" - I don't see the tables being created - it says that the database already exists (it's a VPS i had to rebuild it due to failed installations earlier). What is more I see this warning: 

WARN[0000] /opt/postal/install/docker-compose.yml: `version` is obsolete - could this be causing something?

I created the user and tried to log in anyways but I just see the error mentioned in the beginning.

I decided to follow some other instructions I've found on the web. This time running on Ubuntu 20 - there I could see the tables being created but I was unable to login anyways (might've been something with the DNS).

Since this approach I'm just trying to change some minor things. I don't even know how to do this anymore. 

Anything I might be missing?

Help will be much appreciated.

**Comment by willpower232** [comment]
The version is obsolete message is a feature of newer docker compose so probably not the problem.

What is the output of `postal status`?

**Comment by calybre** [comment]
<img width="674" alt="postal_status" src="https://github.com/postalserver/postal/assets/141501397/e819469b-b49e-40f3-9e32-5f4b64943241">

all up

**Reply by willpower232** [reply]
So they're all up and working and you've apparently been through the database initialisation at least once, have you tried `postal make-user` to create your admin user?

**Reply by calybre** [reply]
yes. 

To make it clear I've gone through the whole pre-setting and installation a couple of times now trying different approaches, right to the end when trying to log in.

So i've created the admin user. But without chance to log in.

**Reply by willpower232** [reply]
What happens when you try to log in? Do you get an error message you can share with a screenshot?

**Reply by calybre** [reply]
I am unable to access Postal.
Can't reach the server - I have the DNS setup with cloudflare - the error message when reaching postal.my_domain.com - "Web server is down". 

When I had it without cloudflare it would just say "Can't reach the website"  -ERR_CONNECTION_TIMED_OUT

Is there a way to access Postal using "server_ip : port" ? When I try this I see "Access denied"

Anything I should check in the config file?

**Reply by willpower232** [reply]
you can try `postal status` but it sounds like something else. Are you definitely running caddy or something else to access the web UI? do you have any firewall in the way of port 80?

**Reply by kayastha1988** [reply]
I got this while accessing to my admin dashboard of postal smtp. On checking log file, got something like this. My login access was correct. Any solution for this issue?

![image](https://github.com/postalserver/postal/assets/19544006/1559ae4c-597f-4a8a-b1fd-bd9356c792c4)


Apr 26 12:14:49 mail sshd[36694]: Failed password for root from 170.64.151.112 port 35878 ssh2
Apr 26 12:14:50 mail sshd[36694]: Connection closed by authenticating user root 170.64.151.112 port 35878 [preauth]
Apr 26 12:14:54 mail sshd[36787]: Invalid user weblogic from 170.64.151.112 port 53768
Apr 26 12:14:55 mail sshd[36787]: pam_unix(sshd:auth): check pass; user unknown
Apr 26 12:14:55 mail sshd[36787]: pam_unix(sshd:auth): authentication failure; logname= uid=0 euid=0 tty=ssh ruser= rhost=170.64.151.112 
Apr 26 12:14:56 mail sshd[36808]: Connection reset by 180.101.88.219 port 47732 [preauth]

**Reply by willpower232** [reply]
Those log entries are likely people trying to SSH into your server as is expected when your SSH port is public.

You will need to do `postal logs web` to see the stack trace.
